src/qibo/tests_new/conftest.py

```diff
@@ -0,0 +1,85 @@
+"""
+conftest.py
+
+Pytest fixtures.
+"""
+import sys
+import pytest
+
+try:
+    import tensorflow as tf
+    _BACKENDS = "custom,defaulteinsum,matmuleinsum,"\
+                "numpy_defaulteinsum,numpy_matmuleinsum"
+    _ENGINES = "numpy,tensorflow"
+    _ACCELERATORS = "2/GPU:0,1/GPU:0+1/GPU:1,2/GPU:0+1/GPU:1+1/GPU:2"
+except ModuleNotFoundError: # pragma: no cover
+    # workflows install Tensorflow automatically so this case is not covered
+    _ENGINES = "numpy"
+    _BACKENDS = "numpy_defaulteinsum,numpy_matmuleinsum"
+    _ACCELERATORS = None
+
+
+def pytest_runtest_setup(item):
+    ALL = {"darwin", "linux"}
+    supported_platforms = ALL.intersection(
+        mark.name for mark in item.iter_markers())
+    plat = sys.platform
+    if supported_platforms and plat not in supported_platforms:  # pragma: no cover
+        # case not covered by workflows
+        pytest.skip("Cannot run test on platform {}.".format(plat))
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "linux: mark test to run only on linux"
+    )
+
+
+def pytest_addoption(parser):
+    parser.addoption("--engines", type=str, default=_ENGINES)
+    parser.addoption("--backends", type=str, default=_BACKENDS)
+    parser.addoption("--accelerators", type=str, default=_ACCELERATORS)
+    parser.addoption("--target-backend", type=str, default="numpy")
```

Comment:
I think this is the only flag which is obscure, could you please add some comment about it?